build.sbt

```diff
@@ -534,6 +534,7 @@ lazy val scioExamples: Project = Project(
     "org.scalacheck" %% "scalacheck" % scalacheckVersion % "test",
     "org.mockito" % "mockito-all" % mockitoVersion % "test"
   ),
+  fork in run := true,
```

Comment:
Is this for everything including `test`?

Reply:
No this only affects `run` and `runMain`.